compose-network/mirror-node/application.yml

```diff
@@ -36,7 +36,7 @@ hedera:
               transferTypes:
                 - CRYPTO
             receiptPercent: 1.0
-            tps: 1.0
+            tps: 10.0
```

Comment:
Why does it need to be higher than 1 TPS? 1 TPS should be sufficient to close the record file every 2s. Any higher creates will eat up too much CPU and disk space for local development.

Reply:
Just testing things out, will change it back to 1.